manifests/init.pp

```diff
@@ -228,7 +225,7 @@
   # catch that error here and notify the user
   $missing_backends = difference($backends, keys($backend_data))
   if count($missing_backends) > 0 {
-    fail("The supplied backends: ${missing_backends} are missing from the backend_options hash:\n ${backend_options}\n
+  fail("The supplied backends: ${missing_backends} are missing from the backend_options hash:\n ${backend_options}\n
```

Comment:
This seems like worse formatting, the indentation doesn't line up.

Reply:
I agree. I just followed the CI (history here https://travis-ci.org/github/voxpupuli/puppet-hiera/pull_requests) until it was all green.

Reply:
I can undo the changes and separate into two different commits, I really don't mind. I sent as one commit because some repos only accept pull requests with one commit. Regardless, it is your call,  I just want to help! =)